docker/types/containers.py

```diff
@@ -115,7 +115,7 @@ def __init__(self, version, binds=None, port_bindings=None,
                  device_read_iops=None, device_write_iops=None,
                  oom_kill_disable=False, shm_size=None, sysctls=None,
                  tmpfs=None, oom_score_adj=None, dns_opt=None, cpu_shares=None,
-                 cpuset_cpus=None, userns_mode=None, pids_limit=None,
+                 cpuset_cpus=None, userns_mode=None, uts=None, pids_limit=None,
```

Comment:
can we rename the parameter to `uts_mode` across the board? That way it matches the API name, is more explicit, and if more `uts`-related parameters are introduced in the future, they'll be easier to integrate.

Reply:
Yep, that name it is more sensible. The updated pull request uses the "uts_mode" attribute name as suggested.